rotkehlchen/exchanges/exchange.py

```diff
@@ -473,11 +473,10 @@ def query_income_loss_expense(
 
         return ledger_actions
 
-    def query_history_with_callbacks(
+    def query_history_with_fail_callback(
             self,
             start_ts: Timestamp,
             end_ts: Timestamp,
-            success_callback: ExchangeHistorySuccessCallback,
             fail_callback: ExchangeHistoryFailCallback,
     ) -> None:
         """Queries the historical event endpoints for this exchange and performs actions.
```

Comment:
Edit the docstring to specify that any results of querying are saved in the DB.
Also remove the success callback mention since you removed it.